### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by johangiraud**
Here I made the logo using icomoon. But since you can't have just one glyph with severals colors, its creates 2 glyphs.

capture d ecran 2017-10-27 a 15 51 54

What do you think of that ?

### CODE REVIEW

**Comment by vinyll** [comment]
Can't we just make a plain SVG that would be the actual logo?
icomoon is interesting for icons, but might be inappropriate to use as the actual website logo.

**Comment by johangiraud** [comment]
Ok sure, but do I need to write Cafébabel as a hidden title somewhere ? like : 
`<h1>Cafébabel</h1>` ?
Or is it fine with the svg only like : 
`` ?

**Comment by vinyll** [comment]
I don't have any certitude on that. A lot of reading on the topic: https://www.fastcompany.com/3016894/should-your-tag-be-your-logo

Also consider you probably want the logo as a link to the homepage.

**Comment by johangiraud** [comment]
oui ! what's the dynamic way to write home url ? like `home_url()` for wp

File: styleguide.html [hunk 1]
```diff
@@ -5,15 +5,7 @@
...
-<h1>
-  <a href=#>
-  <span class=icon-logo-cafebabel>
-    <span class=path1></span>
-    <span class=path2></span>
-    <span class="text-hidden">Cafébabel</span>
-  </span>
-  </a>
-</h1>
+<a href="#" class=logo-cafebabel><img src=./img/logo-cafebabel.svg alt="logo cafébabel"></a>
```
Comment (vinyll) [inline styleguide.html]: href=#
is the `class=logo-cafebabel` mandatory?

**Review by vinyll — APPROVED** [review]
Approved with questions

File: styleguide.html (hunk 1 above)
Comment (johangiraud) [reply styleguide.html]: well how do you target the element otherwise ? there will probably be a lot of `` tag inside `<a>` tag in the page, right ?

**Pull request merged**